backend/src/v2/driver/driver_test.go

```diff
@@ -671,3 +671,87 @@ func Test_extendPodSpecPatch_ImagePullSecrets(t *testing.T) {
 		})
 	}
 }
+
+func Test_extendPodSpecPatch_Tolerations(t *testing.T) {
```

Comment:
@droctothorpe as we discussed, we probably want to have a test for the failure case / invalid toleration? What's the behavior for example for a wrong combination of effect and value?

Reply:
Contingent on implementing validation ourselves.